src/pages/onBoarding/index.js

```diff
@@ -0,0 +1,3 @@
+import OnBoarding from './onBoarding.vue'
+
+export default OnBoarding
```

Comment:
This file is created only to write
```
import OnBoarding from '@/pages/onBoarding'
```
instead of
```
import OnBoarding from '@/pages/onBoarding/onBoarding.vue'
```
in router/index.js?

Reply:
Yes